src/xcode/ENA/ENA/Source/Views/ENSetting/ActionDetailTableViewCell.swift

```diff
@@ -48,6 +48,7 @@ class ActionDetailTableViewCell: UITableViewCell, ConfigurableENSettingCell {
 		iconImageView2.image = images(for: state).1
 		actionButton.setTitle(AppStrings.ExposureNotificationSetting.detailActionButtonTitle, for: .normal)
 
+		actionTitleLabel.font = UIFontMetrics.default.scaledFont(for: UIFont.boldSystemFont(ofSize: 22))
```

Comment:
```suggestion
		actionTitleLabel.font = UIFont.preferredFont(forTextStyle: .title)
```

Reply:
Done with https://github.com/corona-warn-app/cwa-app-ios/pull/112/commits/4bbf3f3e49cd921879b33ccb5c710d49b6f53d95 used .title2 to get the right textsize